lib/lobbyist/v2/communication_campaign.rb

```diff
@@ -33,6 +33,10 @@ def self.create(company_id, params)
         create_from_response(post("/v2/campaigns.json", {company_id: company_id, communication_campaign: params}))
       end
 
+      def self.create_branding_campaign(company_id, params={})
```

Comment:
when we add a method to lobbyist we add a test case, there is a documentation in readme file on how to write test cases(### VCR Tests)

Reply:
done.